docs/data_quality.md

```diff
@@ -0,0 +1,114 @@
+# Properties of a High Quality OSV Record
+
+## Version
+
+1.0.0 (SEMVER)
+
+## Purpose
+
+Describe the “good enough” OSV record that will be imported by OSV.dev
+
+### Out of scope
+
+This does not discuss the problem of record bit rot over time, after initial successful import. The problem of continuous revalidation and treatment of records that have been successfully imported will be dealt with separately in .
+
+Deferred to a future iteration: validating the existence of vulnerable functions in the `ecosystem_specific` field, if supplied.
+
+## Audience
+
+1. Current and aspiring OSV record producers
+2. Downstream OSV.dev record consumers
+
+## Rationale
+
+OSV.dev seeks to be an comprehensive, accurate and timely database of known vulnerabilities (that is highly automation friendly). In order to meet this accuracy goal, a quality bar needs to be both defined and sustainably enforced.
+
+## Properties of a High Quality OSV Record
+
+### Valid
+
+As a prerequisite, it is assumed that a record passes [JSON Schema validation](#appendix-a-osv-schema-validation) for the version of the OSV Schema it declares itself to comply with in the `schema_version` field, or 1.0.0 if it does not.
+
+### Precise
+
+A high quality OSV record allows a consumer of that record to be able to answer the following questions in an **automated** way, at scale:
+
+* “Does this vulnerability, as described, impact me?
+  * “What version do I need to upgrade to for it not to impact me?” 
```

Comment:
Does this imply that OSV records with no fix version fail to be high quality?
eg. If an vuln has not been fixed

Reply:
That's a very fair point, and highlights the assumption that _most_ vulnerabilities have a fix available, which statistically, I would say is the case?

Do you think this assumption should be explicitly stated?

Reply:
You're probably right that most vulns do have fixes, but it seems odd to me that a legitimate advisory about some abandoned software cannot be "high quality".

Another case to consider would be an advisory which was published before a fix is available (log4j comes to mind).

Reply:
Example: around a third of RUSTSEC advisories are for unmaintained crates (https://osv.dev/list?q=RUSTSEC&ecosystem=crates.io), which have no fixed version.

Reply:
I agree with @chrisbloom7, it was not my intention to exclude or otherwise define unfixed vulnerabilities as not of high quality. They are a legitimate class of known vulnerability. I have sought to address the valuable feedback here in 247f7712, PTAL.